### ISSUE TITLE
'firstMd5Digest' of undefined / Evaporate

### ISSUE THREAD

**Issue by bagau** [avocode/avocode#1368]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.13.2`
Avocode UID: 136762
```
TypeError: Cannot set property 'firstMd5Digest' of undefined
    at ext (C:\Users\Admin\AppData\Local\avocode\app-2.13.2\resources\app.asar\node_modules\evaporate\evaporate.js:2029:23)
    at extend (C:\Users\Admin\AppData\Local\avocode\app-2.13.2\resources\app.asar\node_modules\evaporate\evaporate.js:2035:5)
    at FileUpload.updateUploadFile (C:\Users\Admin\AppData\Local\avocode\app-2.13.2\resources\app.asar\node_modules\evaporate\evaporate.js:754:18)
    at C:\Users\Admin\AppData\Local\avocode\app-2.13.2\resources\app.asar\node_modules\evaporate\evaporate.js:1391:33
```

**Comment by jakubzitny** [comment]
Hi @bagau, thanks for sharing this with us. Do you have any more info on how did this happen? Did it happen more than once? Also, are you using 32 or 64bit version of Avocode?

**Comment by bagau** [comment]
Do you have any more info on how did this happen?

**Comment by jakubzitny** [comment]
Thanks for the info @bagau. This should be fixed in the latest version of Avocode - `2.13.5`. You should be able to update automatically on Windows.

There are stilll some small issues with uploading a lot of files at the same time. Tt takes a bit longer than it should and we're working on improving that.

Let us know if you have any other issues with Avocode 😉
